src/ChatManager.js

```diff
@@ -24,30 +40,24 @@ function sleep(ms) {
 
 async function ChatManager(party, token, updateUser, updateState) {
 
-  const ADDRESS_BOOK_TEMPLATE = 'Chat.V2:AddressBook'
-  const CHAT_TEMPLATE = 'Chat.V2:Chat'
-  const MESSAGE_TEMPLATE = 'Chat.V2:Message'
-  const SELF_ALIAS_TEMPLATE = 'Chat.V2:SelfAlias'
-  const USER_TEMPLATE = 'Chat.V2:User'
-  const USER_INVITATION_TEMPLATE = 'Chat.V2:UserInvitation'
-  const USER_ACCOUNT_REQUEST_TEMPLATE = 'Chat.V2:UserAccountRequest'
+  const ADDRESS_BOOK_TEMPLATE = 'Chat.V3:AddressBook'
+  const CHAT_TEMPLATE = 'Chat.V3:Chat'
+  const MESSAGE_TEMPLATE = 'Chat.V3:Message'
+  const SELF_ALIAS_TEMPLATE = 'Chat.V3:SelfAlias'
+  const USER_TEMPLATE = 'Chat.V3:User'
+  const USER_INVITATION_TEMPLATE = 'Chat.V3:UserInvitation'
+  const USER_ACCOUNT_REQUEST_TEMPLATE = 'Chat.V3:UserAccountRequest'
 
   const headers = {
     "Authorization": `Bearer ${token.toString()}`,
     'Content-Type': 'application/json'
   }
 
-  const siteSubDomain = (path = '/data/') => {
+  const siteSubDomain = () => {
```

Comment:
👏 much nicer 👏 

1. did the domain layout change?
2. if so, is dablchat v3 backwards compatible across domain layouts?

Reply:
Yup, domain layout changed, and now we support both!